code/modules/clothing/suits/marine_armor.dm

```diff
@@ -37,7 +37,7 @@
 	var/list/armor_overlays
 	actions_types = list(/datum/action/item_action/toggle)
 	flags_armor_features = ARMOR_LAMP_OVERLAY
-	flags_item = SYNTH_RESTRICTED|IMPEDE_JETPACK
+	flags_item = IMPEDE_JETPACK
```

Comment:
What is this doing in this pr ? I dont get the relation between the two feature

Reply:
so fix it ?

Reply:
I guess I could try removing the file.